--- kdesvn-1.2.3/src/kdesvn.desktop.orig	2008-09-24 12:36:04.000000000 +0200
+++ kdesvn-1.2.3/src/kdesvn.desktop	2009-01-28 14:05:55.384562130 +0100
@@ -4,6 +4,7 @@
 GenericName=SVN client
 GenericName[de]=Graphische Oberfläche für SVN
 GenericName[es]=Cliente SVN
+GenericName[pl]=Klient SVN
 GenericName[lt]=SVN klientas
 Exec=kdesvn %i %m -caption "%c"
 Icon=kdesvn
@@ -13,6 +14,7 @@
 Comment[de]=Ein Subversionclient für KDE
 Comment[nl]=Een Subversion-client voor KDE
 Comment[es]=Un cliente de Subversion para KDE
+Comment[pl]=Klient Subversion dla SVN
 Comment[lt]=Subversion klientas skirtas KDE
 Terminal=false
 MimeType=inode/directory;
